prep_data.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# start elasticsearch if it's not already running
```

Comment:
can we avoid running the ES wait script twice by moving the `schema npm run create_index` command to `run_services.sh`?